### PR TITLE
fs/select : Prevent assert when select called after close

### PR DESCRIPTION
**Pull request by Taejun-Kwon**

### CODE REVIEW

File: os/fs/vfs/fs_poll.c [hunk 1]
```diff
@@ -273,36 +273,41 @@ int file_poll(FAR struct file *filep, FAR struct pollfd *fds, bool setup)
 	FAR struct inode *inode;
 	int ret = -ENOSYS;
 
-	DEBUGASSERT(filep != NULL && filep->f_inode != NULL);
+	DEBUGASSERT(filep != NULL);
```
Comment (amogh-samsung) [inline os/fs/vfs/fs_poll.c:276]: I understand the idea
Calling select on a closed file may be a mistake from the app side, we do not need to have an assert failure.
We can just report the error.

But can you help me understand how and why the need to make this change came up?

**Review by amogh-samsung — COMMENTED** [review]
LGTM

File: os/fs/vfs/fs_poll.c (hunk 1 above)
Comment (Taejun-Kwon) [reply os/fs/vfs/fs_poll.c:276]: To prevent unexpected error in product level.

**Review by bobo2108 — APPROVED** [review]
LGTM!

**Review by sunghan-chang — APPROVED** [review]

**Review by ewoodev — COMMENTED** [review]
LGTM

**Review by jylee9613 — COMMENTED** [review]
lgtm

**Pull request merged**